### PR TITLE
Final push to v2.1.0

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -274,6 +274,13 @@ A user can set permissions via `nfs4_setfacl` on a collection or data object if
 - `chmod` is currently implemented as a `NOP` and will return `0`.
 - NFSRODS currently reports disk free (`df -a`) as `0` to avoid being misleading to other programs.
 
+## Limitations
+### Limitation 1: Write operations do not trigger replication on a replication resource
+This is primarily due to NFSRODS being at the mercy of the client-side application. NFSRODS cannot make assumptions about what a client-side application will do.
```
Comment (trel) [inline README.md:279]: i'd add ... "with regards to multiple connections and streams into a particular data object".

File: README.md [hunk 2]
```diff
@@ -274,6 +274,13 @@ A user can set permissions via `nfs4_setfacl` on a collection or data object if
...
+
+See https://github.com/irods/irods/issues/6142 for more details.
+
+**Migitation**: Configure NFSRODS to point at a resource hierarchy that does not contain a replication resource.
```
Comment (trel) [inline README.md:283]: Mitigation...

or... just use 'Workaround'

File: README.md (hunk 1 above)
Comment (korydraughn) [reply README.md:279]: you're saying append to the second sentence, right?

File: README.md (hunk 2 above)
Comment (trel) [reply README.md:283]: consider an additional sentence...

"A delay rule could be configured to replicate any data to the eventual target location (including the replication resource) or the [Storage Tiering Framework](https://github.com/irods/irods_capability_storage_tiering) could be leveraged to provide this more generally."

File: README.md (hunk 1 above)
Comment (trel) [reply README.md:279]: yes... 
"...will do with regards..."

Comment (trel) [reply README.md:279]: singular regard sounds better...

Comment (korydraughn) [reply README.md]: updated.

File: README.md (hunk 2 above)
Comment (korydraughn) [reply README.md]: replaced "Framework" with "Capability".

any thoughts?

Comment (trel) [reply README.md]: good for me.

**Comment by trel** [comment]
Does Mike know about the need for the snapshot?

**Comment by korydraughn** [comment]
I sent him an email about it. I'm waiting on his response.

**Comment by korydraughn** [comment]
added #

**Pull request merged**